### ISSUE TITLE
About training epoch

### ISSUE THREAD

**Issue by AxeBlue** [yan-roo/SpineNet-Pytorch#4]
Are the default training epochs 350? Can you share the curve of loss?

**Comment by yan-roo** [comment]
![image](https://user-images.githubusercontent.com/49684416/96103043-ca6a0480-0f09-11eb-8314-fe92e4e200c7.png)

**Comment by yan-roo** [comment]
According to the original paper, the protocal B model is trained from scratch for 350 epochs.
If you want to do transfer learning for other dataset, you can decrease the training epochs.

[TensorFlow Implement](https://github.com/tensorflow/tpu/blob/master/models/official/detection/configs/spinenet/spinenet49_retinanet.yaml)
[SpineNet paper](https://arxiv.org/abs/1912.05027)
[MMDetection transfer learning LR and epochs setting](https://github.com/open-mmlab/mmdetection/blob/master/configs/_base_/schedules/schedule_1x.py)

**Comment by AxeBlue** [comment]
Thank you very much!
